spec/factories/sdg.rb

```diff
@@ -14,4 +14,12 @@
 
     target { SDG::Target[code.rpartition(".").first] }
   end
+
+  factory :sdg_revision, class: "SDG::Revision" do
+    %i[budget_investment debate legislation_process poll proposal].each do |model|
+      trait :"#{model}_as_relatable" do
```

Comment:
We use `#{model}_comment` for comments and `for_#{model}` for notifications. Maybe we can follow one of those patterns here as well :thinking:.